src/status_im2/contexts/quo_preview/main.cljs

```diff
@@ -67,7 +67,6 @@
     [status-im2.contexts.quo-preview.notifications.toast :as toast]
     [status-im2.contexts.quo-preview.onboarding.small-option-card :as small-option-card]
     [status-im2.contexts.quo-preview.password.tips :as tips]
-    [status-im2.contexts.quo-preview.posts-and-attachments.messages-skeleton :as messages-skeleton]
```

Comment:
Probably outdated, currently just a blank screen. Animated skeleton will be part of new preview screen